src/products/views/ProductVariantCreate.tsx

```diff
@@ -142,6 +143,7 @@ export const ProductVariant: React.FC<ProductVariantCreateProps> = ({
         onBack={handleBack}
         onSubmit={handleSubmit}
         onVariantClick={handleVariantClick}
+        onWarehouseConfigure={() => navigate(warehouseListPath)}
```

Comment:
Shouldn't it point to warehouse creation view?

Reply:
yep, might be better